### PR TITLE
Dub 46

### PR DESCRIPTION
**Pull request by devang-eminds**

### CODE REVIEW

File: .env.production [hunk 1]
```diff
@@ -0,0 +1,3 @@
+PROFILE_BASE_URL = https://api.dsep-bap-api.becknprotocol.io
+BASE_URL = https://api.dsep-bap-client.becknprotocol.io
+BASE_URL_PROFILE = https://api.dsep-bap-api.becknprotocol.io
```
Comment (nikhil2128) [inline .env.production:3]: ENV_NAME is missing

**Review by nikhil2128 — COMMENTED** [review]
all environments url are same?

**Comment by devang-eminds** [comment]
> all environments url are same?

yes as of now its same for all.

File: .env.production (hunk 1 above)
Comment (devang-eminds) [reply .env.production:3]: ENV_NAME was used only for testing purpose in login screen, In my next PR of DUB-50 I've removed that from login screen, so not used any were for now.

**Pull request merged**